bitacross-worker/app-libs/parentchain-interface/src/integritee/mod.rs

```diff
@@ -39,24 +33,14 @@ use itc_parentchain_indirect_calls_executor::{
 };
 use itp_node_api::metadata::NodeMetadataTrait;
 use itp_stf_primitives::traits::IndirectExecutor;
-use itp_types::CallIndex;
 use log::trace;
-use sp_std::vec::Vec;
+use sp_core::crypto::AccountId32;
 
 /// The default indirect call (extrinsic-triggered) of the Integritee-Parachain.
 #[derive(Debug, Clone, Encode, Decode, Eq, PartialEq)]
 pub enum IndirectCall {
 	#[codec(index = 0)]
-	ShieldFunds(ShieldFundsArgs),
-	#[codec(index = 1)]
-	Invoke(InvokeArgs),
-	// Litentry
-	#[codec(index = 6)]
-	UpdateScheduledEnclave(UpdateScheduledEnclaveArgs),
```

Comment:
Does this mean we allow worker to operate without most recent mrenclave ?

Reply:
I wasn't sure, what to include and what not, I'll update it, thanks @kziemianek

Reply:
Updated, Added back Update and Remove, Keeping this discussion open, as I'm not sure what other indirect calls we still need 😅

Reply:
Yea I prefer not checking mrenclave inside the bc-worker - it should have been resolved upon registration. The client can make sure they are talking to the right worker by checking the wss URL and encrypting the request with the public shielding key.

Wdyt?

Even in sidechain mode, the scheduled-enclave impl wasn't intended for enclave logic check, but rather to stall the block production for upgrade.

Reply:
I'll address this in a follow up PR